_data/simple-icons.json

```
@@ -10697,6 +10697,11 @@
             "hex": "5C0D34",
             "source": "https://www.qatarairways.com/press-releases/en-WW/media_kits"
         },
+        {
+            "title": "qbittorrent",
+            "hex": "2F67BA",
+            "source": "https://www.qbittorrent.org"
```

```suggestion
            "source": "https://github.com/qbittorrent/qBittorrent/blob/73028f9930d9ab9d89edb60ed4aedb66d75669d9/src/icons/qbittorrent-tray-light.svg"
```
Since we have the official icon at their GitHub repository and their website doesn't have an SVG at the top, it would be better to change the source link.